build/package/Installer.RPM.targets

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project='Installer.RPM.props'/>
+
+  <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(CLIBuildDll)"/>
+  <UsingTask TaskName="BuildFPMToolPreReqs" AssemblyFile="$(CLIBuildDll)"/>
+
+  <PropertyGroup>
+    <BuildRpmPackage Condition=" '$(IsRPMBasedDistro)' == True and '$(Architecture)' == 'x64' ">True</BuildRpmPackage>
+  </PropertyGroup>
+
+  <Target Name="GenerateRpms"
+          DependsOnTargets="TestFPMTool;BuildRpms;TestSdkRpm"
+          Condition=" '$(BuildRpmPackage)'=='True' "
+          Outputs="@(GeneratedInstallers)"/>
+
+  <Target Name="BuildRpms"
+          DependsOnTargets="GenerateSharedHostRpm"
+          Condition=" '$(BuildRpmPackage)'=='True' and '$(FPMPresent)'=='True' "/>
```

Comment:
I didn't think too much about this case. This is the same behavior for debian -- we check distro first and later check FPM/debuild, if it doesn't exist, we have silent error and produce tarball only.

It does make sense to error out when there is no FPM/debuild. Do you agree? But I think I should send separate pull request to fix deb